ideaforce/src/main/resources/templates/storydetails.html

```diff
@@ -1,40 +1,81 @@
 <!DOCTYPE html>
 <html xmlns:th="http://www.thymeleaf.org"
-      xmlns="http://www.w3.org/1999/xhtml">
-<head>
-    <span th:include="common/header :: common-header"></span>
-    <link rel="stylesheet" type="text/css" th:href="@{/assets/css/index.css}"/>
-    <title>Around The World :: Story Detail</title>
-</head>
-<body>
-<header>
-    <div th:replace="common/header :: header('admin')"></div>
-</header>
-<div class="container main">
-    <h2 class="text-center">Story Details</h2>
-    <h3>Story Title</h3>
-    <p th:text="${storyDetails.title}"></p>
-    <h3>Description</h3>
-    <p th:text="${storyDetails.description}"></p>
-    <h3>Images</h3>
-    <img th:each="image : ${storyPics}" th:src="@{'data:image/png;base64,'+${image}}"/>
-    <br>
-    <br>
-    <h3>Video Link</h3>
-    <a th:href="${storyDetails.video}">Review Video Uploaded</a>
-    <br/>
-    <br/>
-    <h3>Submitted By</h3>
-    <p th:text="${#strings.capitalize(storyDetails.user.firstName) + ' ' + #strings.capitalize(storyDetails.user.lastName) + ' [' + storyDetails.user.email + '] '}"></p>
-    <br/>
-    <br/>
-    <h3>Submitted From</h3>
-    <p th:text="${#strings.capitalize(storyDetails.user.userLocation)}"></p>
-    <br/>
-    <br/>
-    <a class="info" style="padding:2px;" th:href="@{'/approve/'+ ${storyDetails.id}}">Approve</a>
-    <a class="info" style="padding:2px;" th:href="@{'/addcomments/'+ ${storyDetails.id}}">Add Comments</a>
-</div>
-
-</body>
-</html>
\ No newline at end of file
+      xmlns="http://www.w3.org/1999/xhtml" >
+
+    <head>
+        <span th:include="common/header :: common-header"></span>
+        <link rel="stylesheet" type="text/css" th:href="@{/assets/css/index.css}"/>
+
+        <title>Around The World :: Story Detail</title>
+    </head>
+
+    <body>
+        <header>
+            <div th:replace="common/header :: header('admin')"></div>
+        </header>
+        <div class="container main">
+            <div class="card">
+                <h5 class="card-header text-center" th:text="'Story Details : '+${storyDetails.title}+' by '+${#strings.capitalize(storyDetails.user.firstName) + ' ' + #strings.capitalize(storyDetails.user.lastName)}"></h5>
+                <h5 class="card-title">Challenge Name:</h5>
+                <p class="card-text" th:utext="${storyDetails.challengeDetail.name}"></p>
+
+
+
+                <hr>
+
+                    <h5 class="card-title">Description:</h5>
+                    <p class="card-text" th:utext="${storyDetails.description}"></p>
+
+                    <hr>
+
+                        <h5 class="card-title">Images:</h5>
+                        <p class="card-text" >
+                            <div class="gallery">
+                                <figure th:each="challenge : ${storyPics}">
+                                    <img  th:src="@{'data:image/png;base64,' + ${challenge}}" alt=""/>
```

Comment:
Give fixed height and width, probably show thumbnails and show original image on click